tests/testthat/test-utilities.R

```diff
@@ -52,3 +52,12 @@ test_that("Correct behavior of logSafe", {
   expectedOutput <- c(NA_real_, log10(1), log10(5), log10(defEpsilon), log10(defEpsilon), log10(defEpsilon))
   expect_equal(logSafe(inputVector, base = 10), expectedOutput)
 })
+
+test_that("Correct behavior of foldSafe", {
+  defEpsilon <- getOSPSuiteUtilsSetting("LOG_SAFE_EPSILON")
+  inputX <- c(NA, 1, 5, 0, -1, defEpsilon, 5)
+  inputY <- c(1, -1, NA, 0, -1, defEpsilon, 2)
+
+  expectedOutput <- c(NA_real_, 1 / defEpsilon, NA, defEpsilon / defEpsilon, defEpsilon / defEpsilon, defEpsilon / defEpsilon, 5 / 2)
```

Comment:
why is the 1st expected value `NA_real_` but the 3rd just `NA`?

Reply:
Changed the second to `NA_real_` for consistency.